docker/scripts/osie.sh

```diff
@@ -130,17 +130,29 @@ if ! [[ -f /statedir/disks-partioned-image-extracted ]]; then
 	mkdir $assetdir
 	echo -e "${GREEN}#### Fetching image (and more) via git ${NC}"
 
-	# config hosts entry so git-lfs assets are pulled through our image cache
-	githost="images.packet.net"
-	images_ip=$(getent hosts $githost | awk '{print $1}')
+	# config hosts entry so git-lfs assets from github and our github-mirror are
+	# pulled through our image cache
+	images_ip=$(getent hosts images.packet.net | awk '{print $1}')
 	cp -a /etc/hosts /etc/hosts.new
-	echo "$images_ip        github-cloud.s3.amazonaws.com" >>/etc/hosts.new && cp -f /etc/hosts.new /etc/hosts
+	{
+		echo "$images_ip        github-cloud.s3.amazonaws.com"
+		echo "$images_ip        github.com"
+		echo "$images_ip        github-mirror.packet.net"
```

Comment:
This can be just one line right?

Reply:
I seem to recall having adopted this formatting due to complains from shfmt. To otherwise do this with a single line would probably be an excessively long line.